core/docs/build.md

```diff
@@ -58,7 +64,7 @@ make build_unix
 Now you can start the emulator:
 
 ```sh
-./emu.sh
+bash ./emu.sh
```

Comment:
Why is this required? On my NixOS system this works fine.

Reply:
Even without using patchShebangs? Interesting, I just get bad interpreter.

Reply:
I wonder whether we need bash actually. Maybe it would be better to get rid of any bashisms for the script and change the shebang to `#/bin/sh`. What do you think?

Reply:
Removed bashisms from emu.sh in 917f5c6534e5089c8864513b86cabdc2b402f742